pkg/plugin/v1/scaffolds/internal/templates/makefile.go

```diff
@@ -61,7 +59,7 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
-all: manager
+all: docker-build docker-push install deploy
```

Comment:
The `all` target is typically just for building/compiling (see https://www.gnu.org/software/make/manual/html_node/Standard-Targets.html)

In our case, we don't have code to compile, so I think the next closest thing is just `docker-build`.

```suggestion
all: docker-build
```